src/libcore/num/mod.rs

```diff
@@ -3595,20 +3595,12 @@ impl fmt::Display for TryFromIntError {
     }
 }
 
-#[unstable(feature = "try_from", issue = "33417")]
-impl From<Infallible> for TryFromIntError {
```

Comment:
Maybe leave this as `impl From<!> For TryFromIntError`?  This is here so that you can use `u32: TryFrom<u16>` in a method returning `Result<_, TryFromIntError>`, which I think is still wanted.

Reply:
I removed it thinking that we'd have `impl<T> From ! for T`, but since we won't have that until we have specialization with intersection impls I guess I'd better add it back.